plugins/models/autoencoder.py

```diff
@@ -0,0 +1,48 @@
+import keras
+from keras.layers import *
+from keras.models import Model, Sequential
+from keras.optimizers import Adam
+
+from deepprofiler.learning.model import DeepProfilerModel
+
+
```

Comment:
Great job implementing this autoencoder. We need to find a paper to link here too. Also, although the code is pretty much self-explanatory, try adding some comments to structure code in blocks that humans can read.